ElementX/Sources/Services/Timeline/TimelineProxy.swift

```diff
@@ -77,7 +77,8 @@ final class TimelineProxy: TimelineProxyProtocol {
     
     func messageEventContent(for timelineItemID: TimelineItemIdentifier) async -> RoomMessageEventContentWithoutRelation? {
         guard let content = await timelineProvider.itemProxies.firstEventTimelineItemUsingStableID(timelineItemID)?.content,
-              case let .message(messageContent) = content else {
+              case let .msgLike(content) = content,
+              case let .message(messageContent) = content.kind else {
```

Comment:
May as well nest these here :)
```suggestion
              case let .msgLike(.message(messageContent)) = content,
```

Reply:
Doesn't work like that as I need to switch on the kind